packages/contracts/test/diamond/facets/CreditNftManagerFacet.t.sol

```diff
@@ -12,6 +12,8 @@ import {IERC20Ubiquity} from "../../../src/dollar/interfaces/IERC20Ubiquity.sol"
 import {CreditNft} from "../../../src/dollar/core/CreditNft.sol";
 import {UbiquityCreditToken} from "../../../src/dollar/core/UbiquityCreditToken.sol";
 
+import "forge-std/Test.sol";
```

Comment:
you see how you are reimporting here the Test.sol? this should not be imported directly to files, this should be in the helper file

Reply:
also, why you are touching facets, this PR should be only for core files

Reply:
I had to refactor all tests as part of the spec and other parts of the protocol were failing and had to be refactored too.

Reply:
I see, but make it work from inheritance